File: Pertemuan 6/code/function_m.cpp
#include <bits/stdc++.h>
using namespace std;

int area(int length, int width) {
    return length * width;
    cout << "Hello";
}

void show(int area) {
    cout << "Luas = " << area << endl;
}

void clear() {
    system("cls");
}






// main program
int main() {
    clear();
    int panjang = 4;
    int lebar = 5;
    int luas = area(panjang, lebar);
    show(luas);
    return 0;
}

File: Pertemuan 6/code/function_m.py
from os import system


def area(length, width):
    return length * width



def show(area):
    print("Luas = " + str(area))


def clear():
    system("cls")






# main program

clear()
panjang = 4
lebar = 5
luas = area(4, 5)
show(luas)